crates/speedy-transform/src/web_transform/visit.rs

```diff
@@ -21,7 +21,19 @@ impl Visit for IdentComponent {
     let mut compent_name = match &jsx.opening.name {
       JSXElementName::Ident(ident) => (ident.to_string(), ident.span.ctxt.as_u32()),
       JSXElementName::JSXMemberExpr(member) => {
-        (member.prop.to_string(), member.prop.span.ctxt.as_u32())
+        let mut obj = &member.obj;
+        let real_ident;
+        loop {
+          match obj {
+            JSXObject::JSXMemberExpr(next) => obj = &next.obj,
```

Comment:
Since this would satisfy the case like this `Radio.RadioGroup.RadioItem` (even this is not exist anyway), maybe you can still add a case for that behavior

Reply:
fix